src/jormungandr/testing/jormungandr-automation/resources/explorer/graphql/allblocks.graphql

```diff
@@ -3,160 +3,22 @@ query AllBlocks($last: Int!) {
-                    coefficient
-                    certificate
-                    perCertificateFees {
-                        certificatePoolRegistration
-                        certificateStakeDelegation
-                        certificateOwnerStakeDelegation
-                    }
-                    perVoteCertificateFees {
-                        certificateVotePlan
-                        certificateVoteCast
-                    }}
-    ... on ProposalExpiration{ proposalExpiration }
-    ... on KesUpdateSpeed { kesUpdateSpeed}
-    ... on TreasuryAdd { treasuryAdd }
-    ... on TreasuryParams { treasuryParams {
-                                            fixed
-                                            ratio {
-                                                numerator
-                                                denominator
-                                                }
-                                            maxLimit
-                            }}
-    ... on RewardPot { rewardPot }
-    ... on RewardParams { rewardParams {
-                                    __typename
-                                    ... on LinearRewardParams{
-                                        constant
-                                        ratio {
-                                                numerator
-                                                denominator
-                                            }
-                                        epochStart
-                                        epochRate
-                                    }
-                                    ... on HalvingRewardParams {
-                                        constant
-                                        ratio {
-                                                numerator
-                                                denominator
-                                            }
-                                        epochStart
-                                        epochRate
-                                    }
-                                }}
-    ... on PerCertificateFee{
-            certificatePoolRegistration
-            certificateStakeDelegation
-            certificateOwnerStakeDelegation
-    }
-    ... on FeesInTreasury { feesInTreasury}
-    ... on RewardLimitNone { rewardLimitNone }
-    ... on RewardLimitByAbsoluteStake { rewardLimitByAbsoluteStake {
-                                        numerator
-                                        denominator
-                                    }}
-    ... on PoolRewardParticipationCapping { min max }
-    ... on AddCommitteeId { addCommitteeId}
-    ... on RemoveCommitteeId { removeCommitteeId }
-    ... on PerVoteCertificateFee {
-                                    certificateVotePlan
-                                    certificateVoteCast
-                                }
-    ... on TransactionMaxExpiryEpochs { transactionMaxExpiryEpochs }
```

Comment:
I think you are merging the old version of this and deleting the new, longer one